### ISSUE TITLE
OpenPosen unknown command line flag 'udp_host'

### ISSUE THREAD

**Issue by sscdemodern** [ZiuTinyat/OpenPoseUnityDemo#1]
Hello!
If I want to start OpenPose with the udp_host flag it says: "OpenPoseDemo.exe --udp_host". Please Help!

**Comment by ZiuTinyat** [comment]
Yes, actually I was using an older version of OpenPose, and added the UDP part by my own. 

OpenPose is trying to merge this part to the main project and it seems not working yet.
